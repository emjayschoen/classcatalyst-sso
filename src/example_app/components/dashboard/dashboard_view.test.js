import React from 'react';
import { shallow } from 'enzyme';
import DashboardView from './dashboard_view';

describe('<DashboardView />', () => {
  let subject;
  beforeEach(() => {
    subject = shallow(<DashboardView />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
