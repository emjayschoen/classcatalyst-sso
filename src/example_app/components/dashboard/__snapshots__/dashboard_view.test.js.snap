// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardView /> renders correctly 1`] = `
<styled.div>
  <span>
    DASHBOARD VIEW
  </span>
</styled.div>
`;
